ELS-FiveM/server/server.lua

```diff
@@ -2,28 +2,49 @@ vehicleInfoTable = {}
 patternInfoTable = {}
 
 local verFile = LoadResourceFile(GetCurrentResourceName(), "version.json")
-local curVersion = tonumber(json.decode(verFile).version) or 000
+local curVersion = 0
+if verFile then
+	local data = json.decode(verFile)
+	if data then
+		curVersion = data.version
```

Comment:
Error inducing because `gsub` returns amount replaced.

```suggestion
        curVersion = data.version:gsub("%.", "")
        if tonumber(curVersion) then
```